app/Enums/Prefectures.php

```diff
@@ -0,0 +1,109 @@
+<?php declare(strict_types=1);
+
+namespace App\Enums;
+
+use BenSampo\Enum\Enum;
+
+final class Prefectures extends Enum
```

Comment:
```suggestion
final class Prefecture extends Enum
```
クラス名は単数形で定義します。
https://qiita.com/hironori_narita/items/4b06db0953053d41c4a0

Reply:
参考サイトありがとうございます。
単数形で修正致しました。ご確認宜しくお願い致します。
app/Enums/Prefecture.php